gateway-test/src/test/java/org/apache/knox/gateway/GatewayAdminFuncTest.java

```diff
@@ -159,4 +160,81 @@ public void testAdminService() throws ClassNotFoundException {
 
     TestUtils.LOG_EXIT();
   }
+
+  @Test(timeout = TestUtils.MEDIUM_TIMEOUT)
+  public void testAliasAPI() {
+    String username = "guest";
+    String password = "guest-password";
+
+    String addAliasUrl = clusterUrl + "/api/v1/aliases/test-cluster/myalias";
```

Comment:
Seems weird to be hardcoded here and in the AliasResource class. Can there be a constant for this?

Reply:
good catch, this can be a constant.